src/main/java/com/dawkastresu/medical_clinic/UserController.java

```diff
@@ -0,0 +1,42 @@
+package com.dawkastresu.medical_clinic;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/patients/")
+public class UserController {
+
+    private final PatientRepository patientRepository;
```

Comment:
Utrzymujemy architekturę trójwarstwową ->
Czyli mamy podział na serwisy, repozytoria oraz kontrollery.

Kontrollery -> powinny wykorzystywać serwisy -> Bo ich zadaniem jest odebranie requestu i przekazanie go do odpowiedniej metody z serwisu, która jest odpowiedzialna za realizację logiki biznesowej tego zapytania. Czyli to serwis jesli w ramach wykonywania logiki potrzebuje pobrać dane to pobiera je z repozytorium. Warstwa Controllerów na ten moment powinna zawierać tylko serwisy.